pandas/tests/series/test_constructors.py

```diff
@@ -2096,15 +2077,15 @@ def test_constructor_no_pandas_array(self, using_array_manager):
 
     @td.skip_array_manager_invalid_test
     def test_from_array(self):
-        result = Series(pd.array(["1H", "2H"], dtype="timedelta64[ns]"))
+        result = Series(pd.array([1, 2], dtype="timedelta64[ns]"))
```

Comment:
why do these need to change?

Reply:
This goes through `array_to_timedelta64` seems not to handle string arguments if unit is specified. I suppose this makes sense since "1H" conflicts in unit with ns in the dtype